pylint/checkers/refactoring.py

```diff
@@ -1194,6 +1194,71 @@ def _check_consistent_returns(self, node):
             return
         self.add_message("inconsistent-return-statements", node=node)
 
+    def _is_if_node_return_ended(self, node):
```

Comment:
I think we should add typing on new functions.

```suggestion
    def _is_if_node_return_ended(self, node: astroid.If) -> bool:
```

Reply:
Tanks @Pierre-Sassoulas . You are right, lets use this nice feature!
I have to add it also to other methods. I'll do it ASAP.

Reply:
By the way @Pierre-Sassoulas do you think the `assert(isinstance(node, astroid.If))` worth it or the type hint is enough?
@PCManticore your advice will be appreciated too!

Reply:
I think the typing is enough in this case, although I've argued for the opposite view when building a public API used by non experts. But I suppose pylint's dev are all using typing (they're forced to make mypy happy, at least), so I guess it would be a waste of computation time to do an assert ?

Reply:
@Pierre-Sassoulas totally agree with you! Thanks!